tests/x11/rmt/rmt_export.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: RMT export data
```

Comment:
Acting as rmt support server provide data which used by RMT import client? 
I suppose here we need give more detail for setup.